### ISSUE TITLE
Mobile - List of Chats screen

### ISSUE THREAD

**Issue by phwizard** [dappros/ethora#519]
We need our 'lobby' or List of Chats screen finalized based on your design.

Overall design is good, however see if you could do a better spacing for tabs - perhaps spaced out horizontally like in other design examples.

Also please add a variation with search input and " + New Chat" button which appear if user pulls the list down.

Tabs should be named as following:
Favorites, Groups, Private

<img width="914" alt="Screenshot 2023-11-02 at 14 34 54" src="https://github.com/dappros/ethora/assets/328787/6a43f84c-473b-4ffd-87f0-a8354828e586">

**Comment by polipivnenko** [comment]
![photo_2023-11-09_17-41-39](https://github.com/dappros/ethora/assets/149969403/a13f846d-0168-4a66-acd6-a260f36d5c24)

**Comment by dzinzyura** [comment]
Thank you @polipivnenko! looks great.

One more thing we'll need from you is a "Create new chat" screen. 
This is what it looks like for now:

![IMG_9831](https://github.com/dappros/ethora/assets/99117374/f5d09067-c1ff-460f-9403-bc5c74e6e20f)

Please, implement the designs so that @mykolapylypchukk can finalize it from the client side.

**Comment by dzinzyura** [comment]
@mykolapylypchukk you can start implementing the changes from the provided layout for the chat list. After @polipivnenko  sends you the updated "Create a new chat" layout, you can take it for implementation as well.

**Comment by polipivnenko** [comment]
When creating a new chat, don't you need a part with adding participants? Is the name, photo and description enough?

**Comment by dzinzyura** [comment]
@polipivnenko yes, the users can be added / invited via a chat link or chat QR-code directly from the chat.
![Screenshot 2023-11-10 at 10 34 48 AM](https://github.com/dappros/ethora/assets/99117374/2a62432d-4854-4228-bd9f-844396c35532)

**Comment by phwizard** [comment]
@dzinzyura other screens tasks should be created as separate issues, not as comments

**Comment by phwizard** [comment]
> When creating a new chat, don't you need a part with adding participants? Is the name, photo and description enough?

@polipivnenko Room name is enough. Photo and description are optional. Users can be added / invited separately as Vlad has mentioned.

**Comment by dzinzyura** [comment]
The design of these screens has already been implemented. We'll return to UI & UX improvements (namely animations, swipes, etc.) after the main screens of the application are updated

@phwizard
